lisp/forge-topic.el

```diff
@@ -523,6 +524,32 @@ is called and a topic object is returned if available."
 (defvar-keymap forge-topic-labels-section-map
   "<remap> <magit-edit-thing>" #'forge-edit-topic-labels)
 
+(defun forge--insert-workflow-status (topic)
+  (when (forge--childp topic 'forge-pullreq)
+    (let* ((checks (forge-sql
+                    [:select [name conclusion] 
+                             :from workflow
+                             :where (= commit $s1)]
+                    (oref topic head-rev)))
+           (passed (mapcar
```

Comment:
The status should probably be parsed once and be stored in the database, as I did for issues and pull-requests, I believe.